test/VisualizationTests/HelixWatch3DViewModelTests.cs

```diff
@@ -164,16 +164,13 @@ protected void OpenVisualizationTest(string fileName)
             ViewModel.OpenCommand.Execute(relativePath);
         }
 
-        // With version 2.5 NUnit will call base class TearDown methods after those in the derived classes
-        [TearDown]
-        public void CleanUp()
+        public override void TearDown()
```

Comment:
Does it not need the attribute anymore?

Reply:
it does not since it overrides another method marked as TearDown - at least in nunit2 - I will wait for the tests here to make sure.